flows/codelingo/rewrite/rewrite/writer.go

```diff
@@ -209,6 +212,11 @@ func newFileSRC(ctx *cli.Context, hunk *rewriterpc.Hunk, fileSRC []byte) ([]byte
 	return rewrittenFile, c, nil
 }
 
+// TODO: add review/rewrite comment to template
+func fullSuggestion(newContent string) string {
+	return fmt.Sprintf("```suggestion\n%s\n```\nEdit the suggestion to use an alternative", newContent)
```

Comment:
It's not great that ```suggestion is hard coded here - as it's github specific.

Reply:
I can just remove this again and let Shane deal with it when building github comments.